recipeers/recetas_tradicionales.py

```diff
@@ -0,0 +1,3 @@
+#aqui se genera un set con los ingredientes de la cocina tradicional española
+ingredientes_tradicionales = set()
```

Comment:
¿Esto es parte de qué?
Parte del modelo el problema es indicar la relación entre los diferentes objetos que tengáis.